actions/global_actions.tsx

```diff
@@ -191,22 +191,24 @@ export function showMobileSubMenuModal(elements: any[]) { // TODO Use more speci
     dispatch(openModal(submenuModalData));
 }
 
-export function sendEphemeralPost(message: string, channelId?: string, parentId?: string, userId?: string): void {
-    const timestamp = Utils.getTimestamp();
-    const post = {
-        id: Utils.generateId(),
-        user_id: userId || '0',
-        channel_id: channelId || getCurrentChannelId(getState()),
-        message,
-        type: PostTypes.EPHEMERAL,
-        create_at: timestamp,
-        update_at: timestamp,
-        root_id: parentId,
-        parent_id: parentId,
-        props: {},
-    };
-
-    dispatch(handleNewPost(post));
+export function sendEphemeralPost(message: string, channelId?: string, parentId?: string, userId?: string): ActionFunc {
+    return (doDispatch: DispatchFunc) => {
+        const timestamp = Utils.getTimestamp();
+        const post = {
+            id: Utils.generateId(),
+            user_id: userId || '0',
+            channel_id: channelId || getCurrentChannelId(getState()),
```

Comment:
Minor nit, this should call a `doGetState` so we're not accessing either global function